test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Wide.Tests.ps1

```diff
@@ -15,6 +15,8 @@ Describe "Format-Wide" -Tags "CI" {
 
     It "Should be able to use the autosize switch" {
         { Get-ChildItem | Format-Wide -Autosize } | Should -Not -Throw
+        # Test for #6471
+        { Get-ChildItem | Format-Wide -Autosize | Out-String } | Should -Not -Throw
```

Comment:
What is current directory empty - do we get an exception without the fix?

Reply:
Other tests are also invoked on the premise that there are files in the folder.
I think that it is probably okay.

Reply:
/cc @adityapatwardhan @JamesWTruher Can you approve the test?

Reply:
I think we should make this more predictable and consistent.  How about `Get-ChildItem $TestDrive` (and create at least one file there).  And since the test is for formatting and not testing `Get-ChildItem`, I would suggest doing that call once in `BeforeAll` and saving it to be used in the other tests.